azure-mgmt-compute/src/main/java/com/microsoft/azure/management/compute/implementation/AvailabilitySetsImpl.java

```diff
@@ -71,13 +71,13 @@ public AvailabilitySet typeConvert(AvailabilitySetInner availabilitySetInner) {
     }
 
     @Override
-    public AvailabilitySet get(String groupName, String name) throws CloudException, IOException {
+    public AvailabilitySetImpl get(String groupName, String name) throws CloudException, IOException {
         ServiceResponse<AvailabilitySetInner> response = this.client.get(groupName, name);
         return createFluentModel(response.getBody());
     }
 
     @Override
-    public AvailabilitySet.DefinitionBlank define(String name) {
+    public AvailabilitySetImpl define(String name) {
```

Comment:
Why this change? Returning an `AvailabilitySetImpl` exposes all methods to the user at once.